src/test/kotlin/org/onflow/flow/sdk/FlowAccessApiTest.kt

```diff
@@ -215,32 +215,138 @@ class FlowAccessApiTest {
         val type = "eventType"
         val ids = setOf(FlowId("01"), FlowId("02"))
         val eventResults = listOf(FlowEventResult.of(Access.EventsResponse.Result.getDefaultInstance()), FlowEventResult.of(Access.EventsResponse.Result.getDefaultInstance()))
-        `when`(flowAccessApi.getEventsForBlockIds(type, ids)).thenReturn(eventResults)
+        `when`(flowAccessApi.getEventsForBlockIds(type, ids)).thenReturn(FlowAccessApi.AccessApiCallResponse.Success(eventResults))
 
         val result = flowAccessApi.getEventsForBlockIds(type, ids)
 
-        assertEquals(eventResults, result)
+        assertEquals(FlowAccessApi.AccessApiCallResponse.Success(eventResults), result)
     }
 
     @Test
     fun `Test getNetworkParameters`() {
         val flowAccessApi = mock(FlowAccessApi::class.java)
         val chainId = FlowChainId.TESTNET
-        `when`(flowAccessApi.getNetworkParameters()).thenReturn(chainId)
+        `when`(flowAccessApi.getNetworkParameters()).thenReturn(FlowAccessApi.AccessApiCallResponse.Success(chainId))
 
         val result = flowAccessApi.getNetworkParameters()
 
-        assertEquals(chainId, result)
+        assertEquals(FlowAccessApi.AccessApiCallResponse.Success(chainId), result)
     }
 
     @Test
     fun `Test getLatestProtocolStateSnapshot`() {
         val flowAccessApi = mock(FlowAccessApi::class.java)
         val snapshot = FlowSnapshot("snapshot".toByteArray())
-        `when`(flowAccessApi.getLatestProtocolStateSnapshot()).thenReturn(snapshot)
+        `when`(flowAccessApi.getLatestProtocolStateSnapshot()).thenReturn(FlowAccessApi.AccessApiCallResponse.Success(snapshot))
 
         val result = flowAccessApi.getLatestProtocolStateSnapshot()
 
-        assertEquals(snapshot, result)
+        assertEquals(FlowAccessApi.AccessApiCallResponse.Success(snapshot), result)
+    }
+
+    @Test
```

Comment:
For ease of reviewing could you please split net new added methods from the ongoing merge of the exception handling? I could then approve all the exception handling changes that aren't to new additions

Will be requesting Jordan to pay a closer eye now that we're working at this level. He has more knowledge of Go SDK and expectations for it.

Reply:
@franklywatson Just moved the changes involving new methods to: https://github.com/onflow/flow-jvm-sdk/pull/61. Thanks!